spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/TraceKeys.java

```diff
@@ -283,4 +303,96 @@ public void setHeaders(Collection<String> headers) {
 		}
 	}
 
+	/**
+	 * Trace keys related to Hystrix processing
+	 */
+	public static class Hystrix {
+
+		/**
+		 * Prefix for header names if they are added as tags.
+		 */
+		private String prefix = "";
+
+		/**
+		 * Name of the command key
+		 */
+		private String commandKey = "commandKey";
+
+		public String getPrefix() {
+			return this.prefix;
+		}
+
+		public String getCommandKey() {
+			return this.commandKey;
+		}
+
+		public void setPrefix(String prefix) {
+			this.prefix = prefix;
+		}
+
+		public void setCommandKey(String commandKey) {
+			this.commandKey = commandKey;
+		}
+
+	}
+
+	/**
+	 * Trace keys related to async processing
+	 */
+	public static class Async {
+
+		/**
+		 * Prefix for header names if they are added as tags.
+		 */
+		private String prefix = "";
+
+		/**
+		 * Name of the thread that executed the async method
+		 */
+		private String threadNameKey = "thread";
+
+		/**
+		 * Name of the class from which the asynchronous process started
```

Comment:
simple name? fully qualified? is the class the defining class of the async annotated method? if so, maybe say that and see methodNameKey